src/targets/mobile/config.xml

```diff
@@ -13,6 +13,12 @@
     <allow-intent href="geo:*" />
     <preference name="AppendUserAgent" value="io.cozy.banks.mobile-1.0.4" />
     <hook src="scripts/customizeConfigXML.js" type="before_prepare" />
+    <universal-links>
+        <host name="links.mycozy.cloud">
+            <path event="openUniversalLink" scheme="https" url="/banks/*" />
```

Comment:
This one not. The idea behind is to have : 
links.mycozy.cloud/{slug} 

The 'event' is the event to listen.